### PR TITLE
[Parent Flutter][MBL-13849] Use light/dark mode panda illustrations

### PR DESCRIPTION
**Pull request by JordanMarshall**
This also removes the 'Appearance' section header from the settings screen until there are other section types

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019447&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019454&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/screens/settings/settings_screen.dart [hunk 1]
```diff
@@ -121,20 +97,20 @@ class _SettingsScreenState extends State<SettingsScreen> {
     String semanticsLabel,
     Widget child,
   }) {
-    double size = 130; // May change once placeholders are replaced with panda images
+    double size = 140; // May change once placeholders are replaced with panda images
```
Comment (joehoag) [inline apps/flutter_parent/lib/screens/settings/settings_screen.dart]: Get rid of comment?

**Pull request merged**